operations/rollout-operator-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,50 @@
+local utils = import 'mixin-utils/utils.libsonnet';
+
+(import 'alerts-utils.libsonnet') {
+
+  local alertGroups = [
+    {
+      name: 'rollout_operator_alerts',
+      rules: [
+        {
+          alert: $.alertName('IncorrectWebhookConfigurationFailurePolicy'),
+          expr: |||
+            count by(type, webhook) (kube_validating_webhook_failure_policy{policy="Ignore", webhook=~"^(pod-eviction|zpdb-validation)%s"} > 0)
+          ||| % $._config.webhook_domain(),
+          'for': '1h',
```

Comment:
This seems like a long period of time - should this be shorter?

Anyone who is expecting the webhooks to be disabled could temporarily silence the alert.